compiler/sem/lambdalifting.nim

```diff
@@ -148,10 +148,10 @@ const
   paramName* = ":envP"
   envName* = ":env"
 
-proc newCall(a: PSym, b: PNode): PNode =
-  result = newNodeI(nkCall, a.info)
-  result.add newSymNode(a)
-  result.add b
+proc newObjConstr(typ: PType, info: TLineInfo): PNode =
+  ## Creates an object constructor node with no arguments for type `typ`, using
+  ## `info` for source line information.
```

Comment:
```suggestion
  ## Creates an object construction node with no arguments for type `typ`,
  ## using `info` for source line information.
```

Very soft suggestion to clarify that it's a construction call/expression as opposed to a constructor definition.

Reply:
Yep, that's better, thank you. I think being precise here is important.